docs/contribute/ci-server-setup/Elasticsearch-5.md

```diff
@@ -4,7 +4,7 @@ Install and run using docker:
 
 ```
 docker pull docker.elastic.co/elasticsearch/elasticsearch:5.6.9
-docker run -d -name elasticsearch5 -p 9205:9200 -p 9305:9300 -e cluster.name=elasticsearch -e xpack.ml.enabled=false -e xpack.security.enabled=false docker.elastic.co/elasticsearch/elasticsearch:5.6.9 
+docker run -d --name elasticsearch5 -p 9205:9200 -p 9305:9300 -e cluster.name=elasticsearch -e xpack.ml.enabled=false -e xpack.security.enabled=false docker.elastic.co/elasticsearch/elasticsearch:5.6.9 
```

Comment:
The port here is still `9205` and `9305`

Reply:
ahrgh, yes, ok, I'll fix that as well